### ISSUE TITLE
Add Email notification on public server sync

### ISSUE THREAD

**Issue by mikekuzak** [kinvolk/nebraska#354]
Hi, 

Would be nice to get an email notification from the local Nebraska instance when a public sync updated created new packages. 

Thanks

**Comment by joaquimrocha** [comment]
Yes. This is a good idea. Maybe extending notifications to more services, like email, Slack, etc.
